experiments/DaggerInstanceGLA/Manifest.toml

```diff
@@ -0,0 +1,341 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
This Manifest shouldn't get committed to the Repo examples/*/Manifest.toml should go in the gitignore